### PR TITLE
Adding grouping logic to group values by 100s

### PR DESCRIPTION
**Pull request by ecwillis**

### CODE REVIEW

File: example/index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <script src="../bower_components/jquery/dist/jquery.js"></script>
     <script src="../bower_components/ga-track/dist/ga-track.js"></script>
     <script src="../bower_components/cookie-monster/dist/cookie-monster.js"></script>
-    <script src="../dist/ga-track-fold.bower.js"></script>
+    <script src="../dist/ga-track-fold.js"></script>
```
Comment (jgallen23) [inline example/index.html:6]: why did this change?  we don't need the full version

**Comment by jgallen23** [comment]
- [x] won't auto merge
- [x] don't check in dist files into pull reqs
- [x] you have console.logs

**Comment by ecwillis** [comment]
Removed the `grunt dist` files, but added the files generated by `grunt dev` back in. Fixed conflicts and the `console.logs`

**Comment by jgallen23** [comment]
can you rebase this and just commit the change to the lib file?

**Comment by ecwillis** [comment]
@jgallen23 Does this work?

**Comment by jgallen23** [comment]
YEP!  you know how to tag and release?

**Comment by ecwillis** [comment]
I know how to do git tags ... but I have never pushed anything to bower or npm :) @jgallen23

**Comment by jgallen23** [comment]
released as 0.1.0

**Pull request merged**